app/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/onboarding/OnboardingAuditService.scala

```diff
@@ -163,7 +163,7 @@ class OnboardingAuditServiceImpl @Inject()(auditConnector: AuditConnector) exten
     writes: Writes[A]
   ): Unit = {
     val extendedDataEvent = ExtendedDataEvent(
-      auditSource = "cgt-property-disposals",
+      auditSource = "capital-gains-tax-uk-property",
```

Comment:
can we leave this as it was? We've already agreed the source name with TxM and I'd like to deviate from what we agreed as little as possible. The confluence guidance has this to say about the audit source:
```
Defaults to the name of the play micro-service application but can be set explicitly. Recommend that the auditSource be given a name that will not change with the evolution of the the micro-service.
```

Reply:
reverted